### PR TITLE
Fixes location finder status bar

### PR DESCRIPTION
**Pull request by burkeshartsis**
## TODO:

- [ ] PR has a relevant title that will be understandable in a public changelog (ie...non developers)
- [ ] Closes [tag-issues-here]
- [ ] No new warnings in tests, in storybook, and in-app
- [ ] Upload GIF(s) of iOS and Android
- [ ] Set a relevant reviewer

## REVIEW:

**Manual QA**

- [ ] Manual QA on iOS and ensure it looks/behaves as expected
- [ ] Manual QA on Android and ensure it looks/behaves as expected

**Code Review: Questions to consider**

- [ ] Read through the "Files changed" tab _very carefully_
- [ ] Edge cases: what assumptions are made about input?
- [ ] What kind of tests could be written?
- [ ] How might we make this easier for someone else to understand?
- [ ] Could the code be simpler?
- [ ] Will the code be easy to modify in the future?
- [ ] What's one part of these changes that makes you excited to merge it?

> The purpose of PR Review is to _improve the quality of the software._

### CODE REVIEW

File: packages/WillowCreekApp/src/user-settings/Locations/index.js [hunk 1]
```diff
@@ -74,6 +78,14 @@ class Location extends PureComponent {
     );
   }
 
+  // eslint-disable-next-line class-methods-use-this
+  componentDidUnmount() {
+    /* We are forced to do manually switch back to `light-content` because this renders in a
+     * ModalView so it's the only way to override these styles. It is admitidly brittle. If you
+     * change anything related to status bar style be careful. */
+    StatusBar.setBarStyle('light-content');
```
Comment (conrad-vanl) [inline packages/WillowCreekApp/src/user-settings/Locations/index.js]: So the only problem with this, and I'm really not sure of a fix...but we open this screen from two places - one on the home screen where the status bar style is `light-content`, and one from the settings screen, wehre the status-bar style already is `dark-content`. I think this would effectively set the status bar back to light when you close it which would cause issue on the settings screen

Comment (burkeshartsis) [reply packages/WillowCreekApp/src/user-settings/Locations/index.js]: So I checked and it actually works ok! I was worried about that too. Feel free to test it.

**Comment by burkeshartsis** [comment]
@conrad-vanl @vinnyjth is there a reason we can't review and merge this?
